cub/benchmarks/bench/merge_sort/keys.cu

```diff
@@ -127,7 +127,7 @@ void keys(nvbench::state& state, nvbench::type_list<T, OffsetT>)
   thrust::device_vector<nvbench::uint8_t> temp(temp_size);
   auto* temp_storage = thrust::raw_pointer_cast(temp.data());
 
-  state.exec(nvbench::exec_tag::no_batch, [&](nvbench::launch& launch) {
+  state.exec(nvbench::exec_tag::sync, [&](nvbench::launch& launch) {
```

Comment:
question: why this change is needed? Did merge become synchronous?

Reply:
Yes. Merge sort now has calls to `cudaGridDependencySynchronize`, which cause the benchmark to crash if I were to use `no_batch`.